src/Uno.Toolkit.UI/Behaviors/ItemsRepeaterExtensions.cs

```diff
@@ -454,17 +491,19 @@ private static bool TrySetDefaultSelection(ItemsRepeater ir)
 		{
 			if (ir.ItemsSourceView is { Count: > 0 } isv)
 			{
-				return SetSelectionStates(ir, indexes: 0);
+				return SetSelectionStates(ir, 0);
 			}
 		}
 
 		return false;
 	}
 	private static bool SetEmptySelection(ItemsRepeater ir)
 	{
-		return SetSelectionStates(ir, indexes: null);
+		return SetSelectionStates(ir, null);
 	}
-	private static bool SetSelectionStates(ItemsRepeater ir, params int[]? indexes)
+
+	private static bool SetSelectionStates(ItemsRepeater ir, int? index) => SetSelectionStates(ir, index == null ? null : new[] { index.Value });
```

Comment:
```suggestion
	private static bool SetSelectionStates(ItemsRepeater ir, int index) => SetSelectionStates(ir, new[] { index.Value });
```
there is no such thing as null index, only -1